Assets/Scripts/BulletDeath.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BulletDeath : MonoBehaviour
+{
+    public GameObject impactEffect;
+    
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+
+    private void OnCollisionEnter(Collision collision)
+    {
+        GameObject collisionGameObject = collision.gameObject;
+        if (collisionGameObject.tag != "Player")
+        {
+            Despawn();
+        }
```

Comment:
```suggestion
        if (collisionGameObject.tag != "Player")   Despawn();
```

Kan gewoon op 1 line.